src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Draw {
+    void print();
```

Comment:
The method name `print` in an interface named `Draw` can be misleading. If the purpose is to draw something, consider renaming the method to `draw` to match the interface purpose and to be consistent with the checklist's guidance on single responsibility.